db/schema.rb

```diff
@@ -90,13 +89,13 @@
     t.integer "gallery_images_count"
     t.boolean "epub_download_present"
     t.boolean "mobi_download_present"
+    t.integer "status_id"
     t.boolean "print_black_and_white_a4_download_present"
     t.boolean "print_color_a4_download_present"
     t.boolean "print_color_download_present"
     t.boolean "print_black_and_white_download_present"
     t.boolean "screen_single_page_view_download_present"
     t.boolean "screen_two_page_view_download_present"
-    t.integer "status_id"
     t.string "publication_status", default: "draft"
```

Comment:
can `publication_status` be `null: false`per my comment above

Reply:
Yep 👍🏻